eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Epoch.hs

```diff
@@ -289,9 +304,8 @@ epochTransition = do
         , prevDRepsState =
             es'
               ^. esLStateL . lsCertStateL . certVStateL . vsDRepsL
-        , prevCommitteeState =
-            es'
-              ^. esLStateL . lsCertStateL . certVStateL . vsCommitteeStateL
+        , -- committee state snapshot will not contain members that were removed in Enact
+          prevCommitteeState = updatedCommitteeState
```

Comment:
I believe previous implementation was actually correct. We need to retain the committee state from the previous epoch, in order for committee votes to be valid
```suggestion
        , prevCommitteeState =
            es'
              ^. esLStateL . lsCertStateL . certVStateL . vsCommitteeStateL
```

If I am wrong we'll fix it later, since it should match the DRep implementation above (which is currently unused).


In any case we'll reevaluate the whole time travel thing later, especially after #3719 is merged. For now I think it would be best to revert the implementation, even if I am wrong in my understanding of this flow.

Reply:
It might actually not even matter so much, since it's only removing information that shouldn't be used anyway  (since the vote of the removed members shouldn't even get to be looked up in this map), but  I agree, better to do less if unsure and add to it later :+1: